src/task_14_07.py

```diff
@@ -0,0 +1,25 @@
+# Дописать скрипт. Программа принимает имя папки и имя
+# файла с расширением. Создает папку и создает в ней
+# файл. Если расширение файла py - записывает в файл
+# следующее:
+import os
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-fo', '--folder-name', required=True)
+parser.add_argument('-fi', '--file-name', required=True)
+args = parser.parse_args()
+os.makedirs(args.folder_name)
+os.chdir(args.folder_name)
+print('Folder name:', args.folder_name)
+print('File name:', args.file_name)
+with open(args.file_name, 'w') as file:
```

Comment:
`  if '.py' in args.file_name:`
     `with open(args.file_name, 'w') as file:`
         `file.writelines(...`